src/tests/functional/visit_tests.ts

```diff
@@ -80,6 +90,20 @@ export class VisitTests extends TurboDriveTestCase {
       event.preventDefault()
     }, false))
   }
+
+  async setRequestInterceptor() {
+    this.remote.execute(() => {
+      window.Turbo.setRequestInterceptor(async (request) => {
+        request.addHeader("Authorization", "Bearer Test Token")
+      })
+    })
+  }
+
+  async resetRequestInterceptor() {
+    this.remote.execute(() => {
+      window.Turbo.setRequestInterceptor(async (request) => {})
```

Comment:
thoughts on this approach?

```suggestion
      window.Turbo.setRequestInterceptor(null)
```

or

```suggestion
      window.Turbo.clearRequestInterceptor()
```

Reply:
Great idea. I added `window.Turbo.clearRequestInterceptor()` for resseting interceptor.